client/jetpack-connect/index.js

```diff
@@ -44,15 +44,16 @@ export default function() {
 		page.redirect( `/jetpack/connect/store${ params.interval ? '/' + params.interval : '' }` )
 	);
 
+	page( '/jetpack/connect/plans', '/jetpack/connect/store' );
+	page( '/jetpack/connect/plans/:site', siteSelection, controller.plansSelection );
+	page( '/jetpack/connect/plans/:interval/:site', siteSelection, controller.plansSelection );
```

Comment:
Can we combine these two into one route?

`'/jetpack/connect/plans/:interval(yearly|monthly)?/:site'`